spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
   config.alias_example_group_to :describe_recipe, type: :recipe
   config.alias_example_group_to :describe_helpers, type: :helpers
   config.alias_example_group_to :describe_resource, type: :resource
+  config.file_cache_path = '/var/chef/cache'
```

Comment:
So from experience with this. It should be namespaced to ```/var/chef/cache/ark``` or something unique. Otherwise if you're running CI with multiple cookbooks on the same machine, they will end up clashing with each other.